File: VJudge DAY 01 ( C++ )  BRACU ACM Student Chapter/J.c
#include<stdio.h>
int main()
{
    int i, j, count, factorial, sum=1;
    scanf("%d", &count);
    for(i=1; i<=count; i++)
    {
        scanf("%d", &factorial);
        for(j=1; j<=factorial; j++) sum*=j;
        printf("%d\n", sum);
        sum=1;
    }
    return 0;
}

File: BRACU ACM Student Chapter/Online Workshop on Programming Skills DAY 01 ( JAVA ) BRACU ACM Student chapter/J.java

import java.util.Scanner;

public class J{
  
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);
    int count = input.nextInt();
    for(int i=1; i<=count; i++){
      int sum=1, factorial = input.nextInt();
      for(int j=1; j<=factorial; j++) sum*=j;
      System.out.println(sum);
    }
  }
}
  
